packages/amplify-category-api/src/graphql-transformer/transformer-factory.ts

```diff
@@ -231,3 +235,36 @@ const importTransformerModule = (transformerName: string) => {
     throw error;
   }
 };
+
+/**
+ * Use the options generated from 'transformer-options' to create a V2 GraphQL Transform instance
+ * @param opts The options produced by 'transformer-options'
+ * @returns GraphQLTransform A brand new instance of the GraphQL Transform
+ */
+export const buildGraphQLTransformV2 = async (opts: TransformerProjectOptions<TransformerFactoryArgs>): Promise<GraphQLTransform> => {
```

Comment:
nit: Is this method actually building or in other words applying the transformers on the schema? Or just constructing a `GraphQLTransform` object which has another `transform` method inside it. Would consider renaming if it's the later case.

Reply:
Renamed